src/wavm/dynlink.cpp

```diff
@@ -67,4 +89,170 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env, "dlclose", I32, dlclose, I32 handle)
     // Ignore
     return 0;
 }
+
+// ----------------------------------
+// libffi - see https://github.com/faasm/libffi/
```

Comment:
Adding libffi was the original aim of this PR, as it's needed to support ctypes.